### PR TITLE
Add descriptions for OpenNMS issues from 2016

### PR DESCRIPTION
**Pull request by todb-r7**
Thanks @mkienow-r7 for noticing these never got published!

### CODE REVIEW

File: 2016/6xxx/CVE-2016-6555.json [hunk 1]
```diff
@@ -1,18 +1,99 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6555.json:1]: CNA(s) of user (rapid7) do not match owning CNA.

File: 2016/6xxx/CVE-2016-6556.json [hunk 2]
```diff
@@ -1,18 +1,99 @@
 {
```
Comment (cve-team) [inline 2016/6xxx/CVE-2016-6556.json:1]: CNA(s) of user (rapid7) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by todb-r7** [comment]
Oh! I bet these came from before we were a CNA! I'll bug CERT/CC about this and changing ownership and all that.

**Comment by zmanion** [comment]
How do we change owning CNA? CERT/CC relinquishes ownership to Rapid7. Or we can take Rapid7's content and submit it as ours, which seems awkward and sub-optimal.

**Comment by todb-r7** [comment]
I will happily take ownership of these CVE IDs, @cve-team !

**Pull request merged**